evaluation/swe_bench/README.md

```diff
@@ -0,0 +1,145 @@
+# SWE-Bench Evaluation with OpenDevin SWE-Bench Docker Image
```

Comment:
When this doc is finalized, make sure to acknowledge SWE-Bench and provide a link.

It also would be nice if you could indicate which of the code is from SWE-Bench itself, and which is original code.

Reply:
done!